packages/integration/lib/src/core/configurations/feed_config.dart

```diff
@@ -0,0 +1,130 @@
+import 'package:flutter/services.dart';
+import 'package:likeminds_feed_flutter_core/src/core/configurations/feed_builder.dart';
+import 'package:likeminds_feed_flutter_core/src/utils/web/feed_web_configuration.dart';
+import 'package:likeminds_feed_flutter_core/src/views/activity/configurations/config.dart';
+import 'package:likeminds_feed_flutter_core/src/views/compose/configurations/config.dart';
+import 'package:likeminds_feed_flutter_core/src/views/feed/social/configurations/config.dart';
+import 'package:likeminds_feed_flutter_core/src/views/feed/qna/configurations/config.dart';
+import 'package:likeminds_feed_flutter_core/src/views/feedroom/configurations/config.dart';
+import 'package:likeminds_feed_flutter_core/src/views/likes/configurations/config.dart';
+import 'package:likeminds_feed_flutter_core/src/views/media/configurations/config.dart';
+import 'package:likeminds_feed_flutter_core/src/views/notification/configurations/config.dart';
+import 'package:likeminds_feed_flutter_core/src/views/pending_post/configurations/config.dart';
+import 'package:likeminds_feed_flutter_core/src/views/poll/configurations/config.dart';
+import 'package:likeminds_feed_flutter_core/src/views/post/configurations/config.dart';
+import 'package:likeminds_feed_flutter_core/src/views/report/configurations/config.dart';
+import 'package:likeminds_feed_flutter_core/src/views/search/configurations/config.dart';
+
+// export all the configurations
+export 'package:likeminds_feed_flutter_core/src/utils/web/feed_web_configuration.dart';
+export 'package:likeminds_feed_flutter_core/src/views/feed/social/social_feed_screen.dart';
```

Comment:
Where are we exporting rest of the screens from?

Reply:
from `/likeminds_feed_core.dart` file, here only config files are exported, removed the social_feed_screen export as well from here.

Reply:
Sorted